spec/wings/attribute_transformer_spec.rb

```diff
@@ -24,7 +24,7 @@
 
   it "transform the attributes" do
     expect(subject).to include title: work.title,
-                               contributor: work.contributor,
-                               description: work.description
+                               contributor: work.contributor.first,
+                               description: work.description.first
```

Comment:
Looking at GenericWork, it defines `contributor` and `description` without the `multiple:`.  I thought that defaulted to true.  In which case, I would expect these to be multi-valued in the subject.  Am I missing something?

Reply:
what you're missing is that this module only creates an attribute hash, not a valkyrie object. the `Set.of(Anything)` dry-types spec is what casts back to a set (if appropriate).